app/user/user-resolver.js

```diff
@@ -4,7 +4,7 @@ const fetch = require('../util/fetch');
 const getTokenDetails = (jwt) => {
   const bearerJwt = jwt.startsWith('Bearer ') ? jwt : `Bearer ${jwt}`;
 
-  return fetch(`${config.get('idam.base_url')}/details`, {
+  return fetch(`${config.get('idam.base_url')}/o/userinfo`, {
     headers: {
```

Comment:
this should be calling idam-web-public base url, I assume this is idam-api base url

Reply:
But /o/userinfo available on idam-api as well right? Does it cause any issue ?